### DISCUSSION TITLE
Ring alarm Mode Settings

### DISCUSSION THREAD

**Discussion by Dzoker2** [tsightler/ring-mqtt#908] [General Discussion]
Hi there, 
First of all thank you for all your hard work! I was wondering if there is support for Ring mode settings? I'm specifically asking about changing the Entry/Exit delay function. What I'm trying to do is change the Entry/Exit delay based on the time of the day... No Entry/Exit delay at night and the alarm is armed home and let's say 30s Entry delay during day when I'm home and the alarm is armed home?

Thank you

**Comment by tsightler** [comment]
There is no support for changing such settings, only setting mode arm/disarm modes.

**Reply by Dzoker2** [reply]
Thx
